Sources/PointFree/Logger.swift

```diff
@@ -48,3 +104,20 @@ public struct Logger {
     case fatal
   }
 }
+
+import Either
+
+public func logError<A>(
```

Comment:
more of a proof-of-concept, not using it yet. but we could insert this into any `EitherIO` change to get error visibility. would also like to make middleware helpers that could be inserted in.

Reply:
v cool!